File: ipman.shared/Entity/Wager.cs
using ipman.shared.Entity.Join;
using System;
using System.Collections.Generic;
using System.Text;

namespace ipman.shared.Entity
{
    public class Wager: EntityBase
    {
        public Guid UserAccountID { get; set; }
        public UserAccount UserAccount { get; set; }
        public int WagerAmount { get; set; }
        public virtual ICollection<PostWager> PostWagers { get; set; }
    }
}

File: ipman/ClientApp/entity/Wager.ts
import { EntityBase } from "./EntityBase";
import UserAccount from "./UserAccount";
import PostWager from "./join/PostWager";

export default interface Wager extends EntityBase
{
    UserAccountID: string;
    UserAccount: UserAccount;
    WagerAmount: number;
    PostWagers: PostWager[];
}
